openvpn/start.sh

```diff
@@ -4,9 +4,12 @@ if [ "$OPENVPN_PROVIDER" = "BTGUARD" ]
 then
 	echo "VPN PROVIDER: BTGUARD"
 	vpn_provider="btguard"
-else
+else if [ "$OPENVPN_PROVIDER" = "PIA" ]
```

Comment:
Might not be a bad idea to just make `$OPENVPN_PROVIDER` a required input when running the container.